### DISCUSSION TITLE
View Horizonalpodautoscalers resource and ScaledObjects CRD

### DISCUSSION THREAD

**Discussion by paul-enz** [Ramilito/kubectl.nvim#444] [General]
Hi guys, Is it possible now to filter and view these or is it a future feature request? :)

**Comment by Ramilito** [comment]
Hey! You should be able to view all resources (thanks to @mosheavni fallback feature) that exists but if we don't do anything special to those resources then you will only have basic functionality such as describe, edit and delete.
You can see it in action under the demo section in the readme:

![image](https://github.com/user-attachments/assets/6af48744-eae7-4ee2-a0df-31bcc684be31)
